src/pages/receipts/packageReceipt.js

```diff
@@ -646,8 +645,11 @@ const addDefaultDateReceived = (getCurrentDate) => {
   else dateReceivedEl.value = ""
 }
 
-// returns current date in english canada format ("YYYY-MM-DD")
-const getCurrentDate = () => new Date().toLocaleDateString('en-CA');
+// returns current date in default format ("YYYY-MM-DD")
+const getCurrentDate = () => {
+  const currentDate = new Date();
+  return currentDate.getFullYear() + "-" + currentDate.getMonth() + 1 + "-" + currentDate.getDate()
```

Comment:
have we confirmed this will actually give us what we want? when I run this code in Chrome console, this is what I get back:

![image](https://user-images.githubusercontent.com/18222069/220957830-71bd05f4-bafe-49b4-b744-6dd754442ad1.png)

Reply:
<img width="779" alt="Screenshot 2023-02-23 at 10 51 56 AM" src="https://user-images.githubusercontent.com/30497847/220959530-4d4b6680-f4f8-46eb-9049-beaded087a56.png">

^^ gets returned as date
Im returning `currentDate.getFullYear() + "-" + currentDate.getMonth() + 1 + "-" + currentDate.getDate()` instead of `const currentDate = new Date();`

Reply:
that makes sense, but in your screenshot as well as mine the date returned is not correct

Reply:
Oh I see, the month was getting concatenated instead of getting added. Added padding for single digit date & month.